cajita/src/CMakeLists.txt

```diff
@@ -32,11 +31,14 @@ set(HEADERS_PUBLIC
   Cajita_ParticleGridDistributor.hpp
   Cajita_Partitioner.hpp
   Cajita_ReferenceStructuredSolver.hpp
+  Cajita_SparseArray.hpp
+  Cajita_SparseDimPartitioner.hpp
+  Cajita_SparseIndexSpace.hpp
+  Cajita_SparseLocalGrid.hpp
+  Cajita_SparseLocalGrid_impl.hpp
```

Comment:
Did you mean to include the `_impl` header?

Reply:
Yes, we need to install